app/models/post.rb

```diff
@@ -1,3 +1,15 @@
 class Post < ApplicationRecord
-  belongs_to :user
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  after_save :update_post_counter
+
+  def recent_five_comments
+    Comment.last(5)
+  end
```

Comment:
Post model has a method that returns the 5 most recent comments for a given post 👌